### DISCUSSION TITLE
Modded items with Light Component reset their state every load

### DISCUSSION THREAD

**Discussion by drmahorin** [FakeFishGames/Barotrauma#14911] [Code questions and modding support]
Looks like #10704 but for items from mods that have Light Component.
For example (very bad one, but still) in EK Mods for 1.x.x.x file structure looks fine, but Text Display (Long) Light Component can't be turned off persistently.
But on the other hand vanilla Text Display works just fine.
I've checked .xml tree and it looks fine too in mod, comparing to the vanilla one.

**Comment by Ek-Videogames** [comment]
Vanilla items such as the handheld sonar also don't save their "off" status correctly after a save and reload - and will even emit light still when placed inside of a container. Although this could be a different lighting bug than this issue?
